db/projects/neo/asset-gas-logo-full.svg

```diff
@@ -0,0 +1 @@
0,0,1-5.83-4,19.36,19.36,0,0,1-3.94-5.86,17.49,17.49,0,0,1-1.45-7.08,18.51,18.51,0,0,1,5.32-13.1,18.21,18.21,0,0,1,5.86-4A17.93,17.93,0,0,1,1016.91,509.18Zm-.23,3.22a14.21,14.21,0,0,0-10.53,4.56,15.52,15.52,0,0,0-3.07,4.9,15.73,15.73,0,0,0-1.11,5.86,15.23,15.23,0,0,0,1.11,5.74,15.56,15.56,0,0,0,3.1,4.87,15,15,0,0,0,4.68,3.33,13.89,13.89,0,0,0,5.82,1.22,15.89,15.89,0,0,0,6.05-1.15,14.48,14.48,0,0,0,4.9-3.21,15.47,15.47,0,0,0,3.26-4.83,15,15,0,0,0,1.18-6,15.24,15.24,0,0,0-4.44-11,14.61,14.61,0,0,0-4.86-3.21A15.84,15.84,0,0,0,1016.68,512.4Z"/><path class="cls-23" d="M1059.88,515.46h.15a11.64,11.64,0,0,1,10.49-6.28,13.62,13.62,0,0,1,5.67,1,8.81,8.81,0,0,1,3.56,2.95,12.05,12.05,0,0,1,1.88,4.64,31.8,31.8,0,0,1,.54,6.16v21.37h-3.45V524a42.31,42.31,0,0,0-.23-4.48,9.58,9.58,0,0,0-1.11-3.67,6.36,6.36,0,0,0-2.57-2.49,9.53,9.53,0,0,0-4.59-.92,8.67,8.67,0,0,0-8.5,5.05,16.63,16.63,0,0,0-1.46,5,38.78,38.78,0,0,0-.38,5.44v17.46h-3.45V510h3.45Z"/><path class="cls-24" d="M1121.46,509.18a18.75,18.75,0,0,1,7.31,1.42,18.21,18.21,0,0,1,5.9,3.87,18,18,0,0,1,3.95,5.86,19.18,19.18,0,0,1,0,14.63,18,18,0,0,1-3.95,5.85,18.21,18.21,0,0,1-5.9,3.87,18.75,18.75,0,0,1-7.31,1.42,17.7,17.7,0,0,1-7.16-1.46,18.26,18.26,0,0,1-5.82-4,19.21,19.21,0,0,1-4-5.86,17.49,17.49,0,0,1-1.45-7.08,18.51,18.51,0,0,1,5.32-13.1,18.21,18.21,0,0,1,5.86-4A18,18,0,0,1,1121.46,509.18Zm-.23,3.22A14.21,14.21,0,0,0,1110.7,517a15.52,15.52,0,0,0-3.07,4.9,15.73,15.73,0,0,0-1.11,5.86,15.23,15.23,0,0,0,1.11,5.74,15.41,15.41,0,0,0,3.11,4.87,14.86,14.86,0,0,0,4.67,3.33,13.89,13.89,0,0,0,5.82,1.22,15.89,15.89,0,0,0,6-1.15,14.48,14.48,0,0,0,4.9-3.21,15.3,15.3,0,0,0,3.26-4.83,15,15,0,0,0,1.19-6,15.42,15.42,0,0,0-1.19-6.09,15.26,15.26,0,0,0-3.26-4.87,14.61,14.61,0,0,0-4.86-3.21A15.84,15.84,0,0,0,1121.23,512.4Z"/><path class="cls-25" d="M1164.43,514.39h.15a9,9,0,0,1,3.64-3.83,10.4,10.4,0,0,1,5.25-1.38,10.88,10.88,0,0,1,6,1.65,10.52,10.52,0,0,1,3.94,4.79,10.32,10.32,0,0,1,4.18-4.83,11.89,11.89,0,0,1,6.16-1.61,12.4,12.4,0,0,1,5.56,1.07,8.53,8.53,0,0,1,3.33,3,11.87,11.87,0,0,1,1.65,4.44,33.28,33.28,0,0,1,.42,5.44v22.21h-3.45V522.66a32.26,32.26,0,0,0-.23-3.91,8.63,8.63,0,0,0-1-3.25,5.92,5.92,0,0,0-2.29-2.26,8.23,8.23,0,0,0-4-.84,7.88,7.88,0,0,0-7.32,4,11.7,11.7,0,0,0-1.41,4,26.82,26.82,0,0,0-.39,4.56v20.37h-3.44V523.5a30.2,30.2,0,0,0-.27-4.06,10.53,10.53,0,0,0-1.11-3.56,6.69,6.69,0,0,0-2.38-2.52,7.72,7.72,0,0,0-4.13-1,7.22,7.22,0,0,0-4.48,1.3,9,9,0,0,0-2.69,3.29,15.32,15.32,0,0,0-1.3,4.37,31.42,31.42,0,0,0-.34,4.52v19.45H1161V510h3.45Z"/><path class="cls-26" d="M1222.64,510h3.6l15.09,32,13.55-32h3.53L1234.66,566h-3.52l8.5-20.06Z"/><polygon class="cls-27" points="490.79 101.39 490.79 483.69 289.69 388.85 289.69 173.62 490.79 101.39"/><polygon class="cls-28" points="1.75 490.73 198.03 583.3 198.03 202.79 1.75 110.17 1.75 490.73"/><polygon class="cls-29" points="482.74 94.02 485.97 92.86 487.75 92.23 294.17 0.92 292.39 1.54 287.28 3.37 284.87 4.24 9.51 103.13 203.09 194.45 284.87 165.09 289.69 163.35 482.74 94.02"/></g></svg>
```

Comment:
This is not looks like a specific logo for NEO Gas.. its duplicate to asset-neo-**
Better will be to have these images only one time in repository, by put them on project:
`logo-square.svg`
`logo-full.svg`

instead of 
`asset-gas-logo-square.svg`
`asset-gas-logo-full.svg`
`asset-neo-logo-square.svg`
`asset-neo-logo-full.svg`

Reply:
OK